docs/tutorial-part2.rst

```diff
@@ -25,57 +25,71 @@ Users can select the shortened names (aliases) themselves,
 or let the application generate one.
 Expired aliases will not be reusable.
 
-TODO: screenshot
+A screenshot of the application is shown below:
+
+.. figure:: images/erosion_screenshot.*
```

Comment:
interesting, why `.*` here?

Reply:
Just a habit. When I write documents for my course-related work, I generate both HTML and PDF output. The .* lets me use a .png version for HTML and a .eps version for PDF for the same image.